sqlx-core/src/logger.rs

```diff
@@ -122,37 +157,35 @@ impl<'q> QueryLogger<'q> {
                     String::new()
                 };
 
-                if was_slow {
+                self.span.record("summary", summary);
+                self.span.record("db.statement", sql);
```

Comment:
Something else to keep in mind, and may need adjustment here, is that logging the query isn't free.

Its not uncommon for `select` statements to exceed 25KB or 50KB or more, and 100% logging of those will have a materially negative impact on both the running application outputting the logs, and the log mechanism attempting to capture them. 

This is why the current implementation only logs the query when the query is slow.

Reply:
Yes, I'm taking that into account. The revised implementation will allow the span to be leveled individually, and the query will only be recorded if any of the events or spans are within the configured level.